### PR TITLE
osde2e: Added jobs for new `nvidia-gpu-addon`

### PR DESCRIPTION
**Pull request by sagidayan**
Removed old `gpu-operator` job

### CODE REVIEW

**Review by fabiendupont — APPROVED** [review]
/lgtm

File: ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml [hunk 1]
```diff
@@ -4439,6 +4439,86 @@ periodics:
...
+  cluster: build03
+  cron: 0 */12 * * *
+  decorate: true
+  extra_refs:
+  - base_ref: main
+    org: openshift
+    repo: osde2e
+  labels:
+    pj-rehearse.openshift.io/can-be-rehearsed: "false"
+  name: osde2e-stageing-aws-nvidia-gpu-addon
```
Comment (kpouget) [inline ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml:4452]: `stageing` or `staging`?

**Review by fabiendupont — APPROVED** [review]
/lgtm

**Pull request merged**